### ISSUE TITLE
Insecure sugestion while insall or am I wrong?

### ISSUE THREAD

**Issue by jonathanmmm** [e-alfred/ocdownloader#136]
Hi,

Thanks for your great work,
This way it is much easier to download files when on the run or other files.

In the README.md file it says I should put the parameter:
--check-certificate=false
for aria2

But wouldn`t it be more secure to just leave it out? (Default is true and it would use the local CA Certificates if some are installed

> If you build with OpenSSL or the recent version of GnuTLS which has gnutls_certificate_set_x509_system_trust() function and the library is properly configured to locate the system-wide CA certificates store, aria2 will automatically load those certificates at the startup.
https://aria2.github.io/manual/en/html/aria2c.html#cmdoption-check-certificate
(a little bit above)

**Comment by e-alfred** [comment]
Yes, it could probably omitted because Aria2c only runs on localhost anyway. Could you try it out and tell me if it causes any issues?

**Comment by jonathanmmm** [comment]
I left it out already and I could download from valid CAs but some revoked or so still worked (used a website where you could try it). But still better than without. I couldn't download e.g. https://IP-address.

This problem with revoked or so has probably to do with the not updated CA or so.

you can try different links from https://badssl.com
And see if it works or doesn't work.
